lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -217,15 +217,13 @@ namespace :foreman_tasks do
       end
 
       def self.count_duration(task)
-        duration = nil
-        if task.started_at && task.ended_at
-          duration = (task.ended_at - task.started_at).floor
-        elsif task.started_at && task.ended_at.nil?
-          duration = (Time.now - task.started_at).floor
-        else
-          duration = "N/A"
-        end
-        return duration
+        duration = if task.started_at && task.ended_at
```

Comment:
Almost there. Why is this local variable needed?

Reply:
Ah, my bad